.github/workflows/run_linter_and_unit_tests.yml

```
@@ -0,0 +1,40 @@
+name: Run linter and unit tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+jobs:
+  build:
+
+    runs-on: macos-12
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Set Xcode version
+      run: |
+        ls -l /Applications | grep 'Xcode'
+        sudo xcode-select -s /Applications/Xcode_14.0.1.app
+    - name: Lint code using SwiftLint
```

```suggestion

    - name: Lint code using SwiftLint
```
Please insert a line here (helps readability)